### DISCUSSION TITLE
Improve Discord Connection (Priorize Emotes to Sync)

### DISCUSSION THREAD

**Discussion by thekillernacho** [night/betterttv#5900] [Ideas]
Since the increase of emote slots, the Discord Connection feature to import emotes to Discord could use a little work. The biggest issue is that emote slots on BTTV are now way higher than the amount of emotes a normal Discord server allows. The ability to mark a BTTV emote not to be synced and/or pick a sync order of some kind to prioritize the BTTV emotes you actually want to sync to Discord would be appreciated - in its present state, I have to just go back to uploading emotes to Discord manually or it spams my Discord emote slots with a bunch of emotes I didn't want.

**Comment by BradleyjohnsonTV** [comment]
I agree. The ability to pick what emotes we want to sync with discord would definitely be great.

**Comment by susyemeralds** [comment]
OMG yes please... The fact that BTTV uploads directly to Discord is its best feature but God is it complicated to organise those emotes. There should definitely be a way to choose whether or not each emote is uploaded to Discord.

**Comment by kozaka-tv** [comment]
Had the same idea just right now...BTTV introduced the Emote Sets. Maybe that would be also a good idea to use for that. Like create an emote set for Discord, and add the emotes you want to synchronize. And on the Connections page, if we could select an emote set to sync, that would be perfect!

**Reply by susyemeralds** [reply]
OMG yes that’s it!! Great idea! This means they should allow emotes to be enabled on the channel and to be in a set at the same time (which is not the case right now).
